frontend/src/Student/pages/Scholarships.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
```

Comment:
The URL path from frontend/src/Student/Page.js routes to this page, which passes the appropriate URL parameters to the underlying components